src/main/java/edu/utdallas/davisbase/representation/InsertCommandRepresentation.java

```diff
@@ -0,0 +1,61 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class InsertCommandRepresentation implements CommandRepresentation {
+  private String command;
+  private String table;
+  private List<String> columns;
+  private List<String> values;
```

Comment:
```suggestion
  private final List<String> values;
```

Also, instead of a list of strings, let's make this a list of structured literal value expression objects. Otherwise, the compiler will just have to parse them into their respective value types all over again.